datasets.py

```diff
@@ -717,6 +720,26 @@ def __init__(self, transform, metric='obj_cnt'):
             random.shuffle(info['alllang'])
             pickle.dump(info, open('dataloader_files/yfcc_anns.pkl', 'wb'))
 
+        # Distinguish between global geography labels vs region geography labels
+        country_check_arr = set()
+        for image_id in self.image_ids:
```

Comment:
I don't think it's necessary to check all image_ids in case this takes a while, just like 5 unique ones or something

Reply:
5 unique countries, not 5 unique image_ids